website_sale_wishlist/__openerp__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
@LeartS following "simple headers" way, these lines
https://github.com/LeartS/e-commerce/blob/8.0-website_sale_wishlist/website_sale_wishlist/__openerp__.py#L4-L23
should be removed